mtest/libmscore/unrollrepeats/updateReference

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
I added this one updateReference script too.  Same reason as the edits - a major convenience when running and updating all the  mtests.  This test was added recently, and the creator did not create an updateReference script.  They should have.  I modified the online mtest instructions to include a note telling users to always create an updateReference script for any new mtest directory.

Reply:
`#!/bin/sh` should be enough and more portable

Reply:
I just changed it.  But I copied this from an existing updateReference script.  I think they're all this way.  See the two I modified above here. I might as well change those too.